spec/cc/analyzer/include_paths_builder_spec.rb

```diff
@@ -261,5 +261,18 @@ module CC::Analyzer
         builder.build(["**/subdir*"]).must_equal(["root_file.rb"])
       end
     end
+
+    describe "when analyzing a single file" do
+      let(:cc_includes) { ["subdir/subdir_file.rb"] }
+
+      before do
+        make_file("root_file.rb")
+        make_file("subdir/subdir_file.rb")
+      end
```

Comment:
I can extract the before block, but I think the `let` has to stay since it's used waaaaay above.